### DISCUSSION TITLE
Requeue & Delete Option For Failed Jobs

### DISCUSSION THREAD

**Discussion by itizarsa** [jobrunr/jobrunr#1011] [General]
Does the OSS version support the Requeue & Delete option for failed jobs? 

In my case, it isn't shown on the dashboard

<img width="1440" alt="SS 3" src="https://github.com/jobrunr/jobrunr/assets/77574889/0325c5f7-4cd3-4611-b7ea-fec31ed3f363">

**Comment by rdehuyss** [comment]
Last time I checked, it did :-). This part isn't covered by automated tests though so we may have made a mistake...

Can you scroll to the right?

**Reply by itizarsa** [reply]
You're a saviour @rdehuyss. The content was overflowing. Scrolling to the right revealed the buttons.

**Reply by auloin** [reply]
Hi @itizarsa, can you share some info about the browser (name, version, os...) you're using to run the dashboard?

**Reply by itizarsa** [reply]
Browser - Chrome

Version - Latest

OS - Mac Os

**Reply by auloin** [reply]
Thanks! The page isn't supposed to overflow. I just checked with one of our examples https://github.com/jobrunr/example-java-mag. Are you able to create a reproducer?

**Reply by itizarsa** [reply]
I'll try to create one. It doesn't overflow all the time though

**Reply by itizarsa** [reply]
@auloin Further checking, it seems if the exception message is really long the page is overflowing

**Reply by auloin** [reply]
Thanks for checking! I'll note this down to fix it in a later version.
